### PR TITLE
Added Comments and Reformatted the code

### PR DESCRIPTION
**Pull request by ibmabhishek**
Please review and merge my code to the main branch....

### CODE REVIEW

File: src/main/java/com/ibm/searchorder/OrderSearchController.java [hunk 1]
```diff
@@ -11,9 +11,6 @@
 import com.ibm.searchorder.service.OrderService;
 import com.ibm.searchorder.entity.Order;
 
-//Rest Controller Present here
-//Comment 1
```
Comment (Maseeh98) [inline src/main/java/com/ibm/searchorder/OrderSearchController.java]: unnecessary comment.

File: src/main/java/com/ibm/searchorder/OrderSearchController.java [hunk 2]
```diff
@@ -11,9 +11,6 @@
 import com.ibm.searchorder.service.OrderService;
 import com.ibm.searchorder.entity.Order;
 
-//Rest Controller Present here
```
Comment (Maseeh98) [inline src/main/java/com/ibm/searchorder/OrderSearchController.java]: nice one!

Comment (ibmabhishek) [reply src/main/java/com/ibm/searchorder/OrderSearchController.java]: Thanks

File: src/main/java/com/ibm/searchorder/OrderSearchController.java (hunk 1 above)
Comment (ibmabhishek) [reply src/main/java/com/ibm/searchorder/OrderSearchController.java]: Removed the comments

**Pull request merged**